src/pages/rent-instrument.js

```diff
@@ -4,8 +4,99 @@ import Markdown from 'react-remarkable'
 import PageTemplate from '../templates/pageTemplate'
 
 const RentInstrumentPage = ({ intl }) => (
-  <PageTemplate title={intl.formatMessage({ id: 'rent-instrument.title' })}>
-    <Markdown>{intl.formatMessage({ id: 'rent-instrument.text' })}</Markdown>
+
+
+<PageTemplate title={intl.formatMessage({ id: 'rent-instrument.title' })}>
+    <Markdown options={{ html: true }}>
+      {intl.formatMessage({ id: 'rent-instrument.text' })}
+    </Markdown>
+
+
+  <ul className="nav nav-tabs mb-3" id="ex1" role="tablist">
+      <li className="nav-item" role="presentation">
+        <a
+          className="nav-link active"
+          id="ex1-tab-1"
+          data-toggle="tab"
+          href="#ex1-tabs-1"
+          role="tab"
+          aria-controls="ex1-tabs-1"
+          aria-selected="true"
+        >{intl.formatMessage({ id: 'rent-instrument.strings-title'})}</a>
+      </li>
+      <li className="nav-item" role="presentation">
+        <a
+          className="nav-link"
+          id="ex1-tab-2"
+          data-toggle="tab"
+          href="#ex1-tabs-2"
+          role="tab"
+          aria-controls="ex1-tabs-2"
+        >{intl.formatMessage({ id: 'rent-instrument.wood-title'})}</a
+        >
+      </li>
+      <li className="nav-item" role="presentation">
+        <a
+          className="nav-link"
+          id="ex1-tab-3"
+          data-toggle="tab"
+          href="#ex1-tabs-3"
+          role="tab"
+          aria-controls="ex1-tabs-3"
+        >{intl.formatMessage({ id: 'rent-instrument.brass-title'})}</a
+        >
+      </li>
+      <li className="nav-item" role="presentation">
+        <a
+          className="nav-link"
+          id="ex1-tab-4"
+          data-toggle="tab"
+          href="#ex1-tabs-4"
+          role="tab"
+          aria-controls="ex1-tabs-4"
+        >{intl.formatMessage({ id: 'rent-instrument.percussion-title'})}</a
+        >
+      </li>
+      <li className="nav-item" role="presentation">
+        <a
+          className="nav-link"
+          id="ex1-tab-5"
+          data-toggle="tab"
+          href="#ex1-tabs-5"
+          role="tab"
+          aria-controls="ex1-tabs-5"
+        >{intl.formatMessage({ id: 'rent-instrument.other-title'})}</a
+        >
+      </li>
+    </ul>
+
+    <div className="tab-content" id="ex1-content">
+      <div className="tab-pane fade show active" id="ex1-tabs-1" role="tabpanel" aria-labelledby="ex1-tab-1">
```

Comment:
It might be a better idea to generate these tabs and their contents from the contents of the YAML file. This prevents future editors of the website text from having to understand React, and instead allows them to configure the contents of this page in the more readable YAML format.

Also I don't like code repetition :p

Reply:
Content is already read from YAML, they just aren't automatically generated yet. But I'm absolutely with you here :)

Reply:
Fair enough! As long as we keep reviewing the changes being made, it indeed is not a very big deal if some of the content is directly in the HTML. And it's a good thing we are a Delft association, which means that plenty of people know how to work with HTML in the first place :)